.github/workflows/release.yml

```diff
@@ -56,15 +56,15 @@ jobs:
         shell: bash
         run: |
           release_name="tcli-${{ steps.tag.outputs.tag }}-${{ matrix.target }}"
-          dotnet publish ThunderstoreCLI/ThunderstoreCLI.csproj -c Release -r "${{ matrix.target }}" -o "$release_name" -p:PublishReadyToRun=true
+          dotnet publish ThunderstoreCLI/ThunderstoreCLI.csproj -c Release -r "${{ matrix.target }}" -o "${release_name}"
```

Comment:
I tried to move more of the configuration switches to the .csproj instead of them being here in the action, though I actually did mess up a bit when fixing up the nuget build down below, should be fixed now.